.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@0.2.1
+
+commands:
+  setup:
+    steps:
+      - checkout
+      - python/load-cache
+      - python/install-deps
+      - run: pip install -r requirements-dev.txt
+      - python/save-cache
+
+jobs:
+  black:
+    executor: python/default
+    steps:
+      - setup
+      - run:
+          command: black --check src test scripts
```

Comment:
I'd like to make this step fix the files and commit it back to the branch, but I'll attempt that in a follow-up